CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ option(INEXOR_BUILD_BENCHMARKS "Build benchmarks" OFF)
 option(INEXOR_BUILD_DOC "Build documentation" OFF)
 option(INEXOR_BUILD_EXAMPLE "Build example" ON)
 option(INEXOR_BUILD_TESTS "Build tests" OFF)
+option(INEXOR_BUILD_WITH_CHECKS "Build tests" OFF)
```

Comment:
Change the description please.

Reply:
`option(INEXOR_BUILD_WITH_CHECKS "Build with compiler flags -Wall and -Wextra" OFF)`